build.gradle

```diff
@@ -53,7 +53,7 @@ ext {
 	]
 
 	// Code Coverage (Lines/Branches) cannot be below this value on a per sub project basis
-	minUnitTestLineCoverage = 0.9
+	minUnitTestLineCoverage = 0.8
```

Comment:
just double checking is 80% agreed with HMRC now?

Reply:
Yeah this is ok just cant drop any more